webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/Application.scala

```diff
@@ -19,10 +18,12 @@ class Application @Inject()(redisClient: DataStoreRedisStore, applicationHealthS
   def health: Action[AnyContent] = Action.async { implicit request =>
     log() {
       for {
-        before <- Fox.successful(Instant.now)
+        before <- Fox.successful(System.currentTimeMillis())
         _ <- redisClient.checkHealth
+        afterRedis = System.currentTimeMillis()
         _ <- Fox.bool2Fox(applicationHealthService.getRecentProblem().isEmpty) ?~> "Java Internal Errors detected"
-        _ = logger.info(s"Answering ok for Datastore health check, took ${Instant.since(before)}")
+        _ = logger.info(
+          s"Answering ok for Datastore health check, took ${afterRedis - before} ms (Redis at ${redisClient.authority} ${afterRedis - before} ms)")
```

Comment:
Your updated version of the logging message includes the time span needed for the health check twice. Is there a specific reason for that?
```suggestion
          s"Answering ok for Datastore health check, took ${afterRedis - before} ms (Redis at ${redisClient.authority})")
```

Reply:
Is it to match the logging schema of the tracing store? If that's the case its fine by me :)